pom.xml

```diff
@@ -139,6 +139,11 @@
                 <enabled>true</enabled>
             </releases>
         </repository>
+        <repository>
+            <id>Shibboleth</id>
+            <name>Shibboleth</name>
+            <url>https://build.shibboleth.net/nexus/content/repositories/releases/</url>
```

Comment:
What is requiring this new repository? I see there is a new dependency for `spring-security-saml2-core` but that appears to be in maven central.

Reply:
Shibboleth is the repo where OpenSAML is released to and it seems like newer OpenSAML released are only in their specific repo and not in Maven central. 

The spring-security-saml project had to do something similar here:
https://github.com/spring-projects/spring-security-saml/blob/master/build.gradle#L50

Reply:
Thanks for the clarification.